server/controllers/memoCtrl.js

```diff
@@ -199,13 +184,83 @@ function markOutdatedMemoDatas(memoDatas = [], roundInfos = []) {
         }
       : {}
   )
+
   const markedOutDatedMemoDatas = memoDatas.map(m => ({
     ...m,
     ...{ outdated: outdatedMemoData(offerings, startSelectionYear, m) },
   }))
+
+  markedOutDatedMemoDatas.forEach(memo => {
+    memo.ladokRoundIds.forEach(roundId => {
+      const roundInfo = roundInfos.find(
+        x => x.round.ladokRoundId === roundId && Number(memo.semester) === x.round.startTerm.term
```

Comment:
ladokRoundIds will disappear with kopps api, it means it will be impossible to do `x.round.ladokRoundId === roundId` in the future. Therefore application code should be added to the admin side and saved in the database. Here application code should be read from `kurs-pm-data-api` and matched to application codes from kopps-api.

Reply:
Ok i got your point. I will update admin side and then update this PR and i will remove round id from here.